plugins/sinks/influxdb.go

```diff
@@ -0,0 +1,83 @@
+package main
+
+import (
+	"encoding/json"
+	api "github.com/emqx/kuiper/xstream/api"
+	_ "github.com/influxdata/influxdb1-client/v2"
+	client "github.com/influxdata/influxdb1-client/v2"
+	"log"
+	"time"
+)
+
+type influxSink struct {
+	url       string
+	table     string
+}
+
+type ListMap []map[string]float64
+
+func (m *influxSink) Configure(props map[string]interface{}) error {
+	return nil;
+}
+
+func (m *influxSink) Open(ctx api.StreamContext) (err error) {
+	logger := ctx.GetLogger()
+	logger.Debug("Opening influx sink")
+	return
+}
+
+func (m *influxSink) Collect(ctx api.StreamContext, data interface{}) error {
+	cli := connInflux();
```

Comment:
Why do we need to setup a new connection everytime? The collect function will be called for each message, is there a problem to reset connection each time like this? And it seems the cli(connection) does not close, will it be a problem? For other sink/source like mqttSink, we only hold a connection throughout the liftcycle. Can we do the same for influxdb?